policyengine_us/parameters/gov/states/ca/tax/income/credits/young_child/loss_threshold.yaml

```
@@ -0,0 +1,15 @@
+description: CA young child tax credit loss threshold.
+values:
+  2021-01-01: 0
+  2022-01-01: 32_490
+
+metadata:
+  unit: currency-USD
+  period: year
+  name: ca_young_child_loss_threshold
+  label: CA young child tax credit loss threshold
```

We're inconsistent about labels right now but I'd like to start spelling them out.

They seem to capitalize the credit in public documents.
```suggestion
  label: California Young Child Tax Credit loss threshold
```

---

Done